net/wireguard/src/opnsense/mvc/app/controllers/OPNsense/Wireguard/Api/ServiceController.php

```diff
@@ -73,4 +73,16 @@ public function showhandshakeAction()
         $response = $backend->configdRun("wireguard showhandshake");
         return array("response" => $response);
     }
+
+    /**
+     * reload wireguard config
+     * @return array
+     */
+    public function reloadAction()
```

Comment:
Well this is fine, but we should really discuss a way for bringing this into the GUI because otherwise the next ticked will be opened asking for it.

Reply:
Agreed. I'd love hear some thoughts on how to message the "this button will only do one specific thing". Happy to make that change!

Reply:
I've added this in the form of a Refresh button, mimicking how the Nginx page works. Let me know if this is sufficient for this PR or if something different should happen.

Reply:
Sorry for radio silence. I've looked at the GUI again.. any reason we can't overlay the "apply" button in the Endpoints/Clients tab? Your button does the same except it does reload instead of restart... so is there any Endpoint change (e.g. adding a new or deleting) that would require a restart?

Reply:
Yes, that's the tricky part. I believe most endpoint changes require a restart, but I'm not 100% sure which changes those are, so this felt safest.